apps/system/js/app_usage_metrics.js

```diff
@@ -847,6 +847,13 @@
       return false;
     }
 
+    // Bug 1134998: Don't track apps that are marked as private windows
+    // Some app-like objects may not have the isPrivateBrowser function,
+    // so we also check to make sure it exists here.
+    if (app.isPrivateBrowser && app.isPrivateBrowser()) {
```

Comment:
This test might be slightly safer written as:

if (typeof app.isPrivateBrowser === 'function' && app.isPrivateBrowser())